client/modules/core/csvtimeline/csvtimeline.component.ts

```diff
@@ -29,16 +29,17 @@ import {
     FormBuilder,
     Validators
 } from '@angular/forms';
-import {
-    suggestionComponent
-} from './suggestoptionComponent/suggestoption.component';
+
 import template from './csvtimeline.html';
 
 
 @Component({
     selector: 'csvtimeline',
     template,
-    directives: [REACTIVE_FORM_DIRECTIVES, suggestionComponent]
+    directives: [REACTIVE_FORM_DIRECTIVES
+    // ,
+    //  suggestionComponent
+     ]
 })
 
 export class CsvTimelineComponent extends MeteorComponent implements OnInit, OnChanges {
```

Comment:
dont use any

data_month: any;
     sort_order: any;
     month_in_headbar: any;
     yearly: any;
     monthly: any;
     dateB: any;
     dbdate: any;


in of creating so many variables create a json model with typescript 
same as Hero object in angular2 tutorial